NtupleTools/python/ntuple_builder.py

```diff
@@ -212,7 +212,7 @@ def make_ntuple(*legs, **kwargs):
             )
 
     # If we have two legs or photons, we are interested in VBF selections.
-    if len(legs) == 2 or 'g' in legs:
+    if len(legs) == 2 or 'g' in legs or hzz:
```

Comment:
I'd prefer just moving this to the main templates. I want these vbf variables, Kenneth wants them, and you want them. That's everybody.